pkg/cmd/experimental/syncgroups/groupsyncer.go

```diff
@@ -39,15 +40,35 @@ type LDAPGroupSyncer struct {
 
 // Sync allows the LDAPGroupSyncer to be a GroupSyncer
 func (s *LDAPGroupSyncer) Sync() []error {
+	openshiftGroups, errors := s.GetResultingGroups()
+
+	for _, openshiftGroup := range openshiftGroups {
+		fmt.Printf("group/%s\n", openshiftGroup.Name)
```

Comment:
> log or was this on purpose?

Intentional.  This produces output that can be piped through xargs to other commands to inspect the resources that have been modified.